test/integration/singlecluster/controller/jobs/job/job_controller_test.go

```diff
@@ -2566,20 +2565,43 @@ var _ = ginkgo.Describe("Job controller when TopologyAwareScheduling enabled", g
 		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
 
 		nodes = []corev1.Node{
-			*testingnode.MakeNode("b1").
+			*testingnode.MakeNode("b1-r1").
```

Comment:
What is the reason for the changes to the integration tests?

Reply:
I'd like to explicitly showcase how the unconstrained (and implict default) works

Reply:
Oh yeah, it makes sense, let me update that then